frontend/src/app/patients/Components/PersonForm.tsx

```diff
@@ -169,6 +171,26 @@ const PersonForm = (props: Props) => {
       if (value === patient[field]) {
         return;
       }
+      if (field === "unknownAddress") {
+        setFormChanged(true);
+
+        //if unknown address set to a valid default; else an empty string
+        const state = value ? "NA" : "";
+        const zip = value ? "00000" : "";
```

Comment:
Can you tell me a little more about why we're using these default values here instead of an empty string or the like?

Reply:
I can't speak to why these default values were already in place (for non USA countries I believe).  But I know the backend validation won't accept an empty string for either of these fields.  And using the default "we don't have a value for that" values seemed safer in terms of making sure this plays nicely with existing behaviors - as well as not having to change the graphql validation.

Reply:
Great, I didn't realize those values were already being utilized on this form. Good enough for me!